packages/allure-js-commons/test/spec/FileSystemAllureWriter.spec.ts

```diff
@@ -39,15 +39,18 @@ describe("FileSystemAllureWriter", () => {
     expect(actualContent.toString("utf8")).toBe(data);
   });
 
-  it("Should create allure-report nested path", () => {
+  it("creates allure-report nested path every time writer write something", () => {
```

Comment:
Given the spec, a proper test would be to delete the directory after `writeTest` and check if it's re-created after another one.

Reply:
👍